Data-Structures/Array/MajorityElement.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * [Majority Element](https://www.geeksforgeeks.org/majority-element/) A majority element in an array A[] of size n is an element that appears more than n/2 times (and hence there is at most one such element)
+ * @complexity: O(n) (on average )
+ * @complexity: O(n) (worst case)
+ * @flow
+ */
+
+// To find the candidate for majority element
+function Candidate (array) {
```

Comment:
Please name this `findCandidate` and remove the comment.

Reply:
Done